src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -424,7 +427,7 @@ const getPublicness = (currentTabName: typeof TABS[number]) => {
       return true
     case 'Unlisted':
       return false
-    case 'All Videos':
+    case 'All videos':
     default:
       return undefined
```

Comment:
Since you went ahead and refactored this to use those `isTab` variables, let's also use them in this `getPublicness` function